### PR TITLE
It7

### PR DESCRIPTION
**Pull request by garcia50**
Here we have it7 model methods, I've also added the some CRUD test for the categories.

### CODE REVIEW

File: app/controllers/little_shop_app.rb [hunk 1]
```diff
@@ -14,6 +14,10 @@ class LittleShopApp < Sinatra::Base
     erb :"merchants/new"
   end
 
+  get 'merchants-dashboard' do
```
Comment (lnchambers) [inline app/controllers/little_shop_app.rb:17]: Do we need merchants in here?

File: app/controllers/little_shop_app.rb [hunk 2]
```diff
@@ -112,8 +120,4 @@ class LittleShopApp < Sinatra::Base
     erb :"/items/show"
   end
 
-  get '/api/v1/items/:id' do |id|
```
Comment (lnchambers) [inline app/controllers/little_shop_app.rb]: This needs to stay in.

File: app/models/merchant.rb [hunk 3]
```diff
@@ -1,4 +1,13 @@
 class Merchant < ActiveRecord::Base
   validates_presence_of :name
+
   has_many :items
+
+  def self.how_many_items?
```
Comment (lnchambers) [inline app/models/merchant.rb]: Were you working on merchants as well?

**Review by lnchambers — CHANGES_REQUESTED** [review]
Look over some of the things. It looks like you did some work on merchants and took out the JSON path. Was there a reason?

File: app/controllers/little_shop_app.rb (hunk 2 above)
Comment (garcia50) [reply app/controllers/little_shop_app.rb]: Thanks for catching that, we definitely need it.

File: app/models/merchant.rb (hunk 3 above)
Comment (garcia50) [reply app/models/merchant.rb]: I wasn't working on merchants, but it looks like I might have added a space in between on accident.

**Review by lnchambers — APPROVED** [review]

**Pull request merged**